app/views/trial_scorings/show.html.haml

```diff
@@ -17,6 +17,10 @@
       %tr
         %th.plant_scoring_units_scoring_unit_name_column
           = t 'tables.plant_scoring_units.scoring_unit_name'
-        - @trait_descriptors.map{ |td| td.try(:trait).try(:name) }.each do |column|
-          %th= column
+        - @trait_descriptors.map{ |td| [td.id, td.try(:trait).try(:name)] }.each do |id, column|
```

Comment:
I know this was not changed now, but it is possible for TD to not have a T, or for a T to not have a name?